python/paddle/fluid/contrib/slim/quantization/quantization_mkldnn_pass.py

```diff
@@ -350,6 +351,8 @@ def _gather_scales(self, graph):
                 use_unsigned_int = False
                 self._var_quant_scales[input_name] = (use_unsigned_int,
                                                       lod_tensor)
+                self._var_quant_scales[scale_name.replace(".scale", "")] = (
```

Comment:
What is this prepare for? Are `input_name` and `scale_name.replace(".scale", "")` different？

Reply:
I wish they were the same, but when we encounter the follwing case:
[op]->[fake_dequantize]->[fake_quantize]->[another_op] 
as it is in, for example, VGG16, the tensor that enters `fake_quantize_moving_average_abs_max` has a `.dequantized` suffix.
![image](https://user-images.githubusercontent.com/4967505/66549578-da574780-eb43-11e9-98af-515bae89e16f.png)
In this example `pool2d_1.tmp_0.dequantized` enters `fake_quantize_moving_average_abs_max`.

Reply:
The reason for which we need the clear names without any *.dequantized, or *.scale suffixes, is that after the graph is cleared off of all the fake-(de)quantize operators, the tensors with suffixes disappear, and `cpu_quantize_pass` cannot find appropriate scales for tensors with normal names, such as `pool2d_1.tmp_0`.

Reply:
A very clear explanation. :thumbsup: